Resources/templates/assets/css/modaal.css

```diff
@@ -0,0 +1,661 @@
+/*!
```

Comment:
Je l'ai enlevé et envoyé une nouvelle Pull Request.

Le 03/06/16 15:26, Godfraind a écrit :
>
> In Resources/templates/assets/css/modaal.css 
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65706148>:
>
> > @@ -0,0 +1,661 @@
> > +/*!
> Il est dans ton commit, il suffit de le supprimer. Sinon je le fais 
> moi même. 2016-06-03 15:18 GMT+02:00 om64 <notifications@github.com>:
> … <#>
> In Resources/templates/assets/css/modaal.css <#14 (comment) 
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65705059>> 
> : > @@ -0,0 +1,661 @@ > +/*! Comment? Il n'est appelé par aucune page 
> du site. Il faut juste éviter de l'uploader. Le 03/06/16 15:12, 
> Godfraind a écrit : … <#m_-4515889241266288425_> In 
> Resources/templates/assets/css/modaal.css <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> (comment) <#14 
> (comment) 
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65704339>>>: 
> > @@ -0,0 +1,661 @@ > +/*! Tu peux le retirer alors ? 2016-06-03 14:59 
> GMT+02:00 om64 ***@***.***>: … <#> In 
> Resources/templates/assets/css/modaal.css <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> (comment) <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> (comment) <#14 
> (comment) 
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65702678>>>> 
> : > @@ -0,0 +1,661 @@ > +/*! Ce fichier a été utilisé pour un mockup: 
> il ne sert à rien pour le site. Le 03/06/16 14:51, Godfraind a écrit : 
> … <#m_-8844681618930724626_> In 
> Resources/templates/assets/css/modaal.css < #14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>> (comment) <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> (comment) <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> (comment) <#14 
> (comment) 
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65701645>>>>>: 
> > @@ -0,0 +1,661 @@ > +/*! C'est à dire ? 2016-06-03 14:48 GMT+02:00 
> om64 ***@***.***>: … <#> In Resources/templates/assets/css/modaal.css 
> <#14 <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>>> (comment) 
> <#14 <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>> (comment) <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> (comment) <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> (comment) <#14 
> (comment) 
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65701151>>>>>> 
> : > @@ -0,0 +1,661 @@ > +/*! Il ne faut pas en tenir compte! Le 
> 03/06/16 14:46, Godfraind a écrit : … <#m_9033775370283028204_> In 
> Resources/templates/assets/css/modaal.css <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>>>> (comment) 
> <#14 <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>>> (comment) 
> <#14 <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>>> (comment) <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14>> (comment) <#14 
> <https://github.com/claroline/NetThemeBundle/pull/14> (comment) <#14 
> (comment) 
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65700972>>>>>>>: 
> > @@ -0,0 +1,661 @@ > +/*! C'est normal le fichier modaal.css ? — You 
> are receiving this because you authored the thread. Reply to this 
> email directly, view it on GitHub < 
> https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65700972>, 
> or mute the thread < 
> https://github.com/notifications/unsubscribe/ANpV2hls_b7kTZLh_rxzVAXYYhycEdyYks5qICI5gaJpZM4ItfhJ>. 
> — You are receiving this because you commented. Reply to this email 
> directly, view it on GitHub < 
> https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65701151>, 
> or mute the thread < 
> https://github.com/notifications/unsubscribe/ABVSttG0RqT1YCivxw6as2iTMi9SQYi-ks5qICKIgaJpZM4ItfhJ> 
> . -- Godfraind Nicolas 5, rue de la Carrière 6592 Monceau-Imbrechies 
> GSM: 0472 94 80 99 — You are receiving this because you authored the 
> thread. Reply to this email directly, view it on GitHub < 
> https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65701645>, 
> or mute the thread < 
> https://github.com/notifications/unsubscribe/ANpV2pqs17V_LzqV_v-t6QT9LpQB683bks5qICNkgaJpZM4ItfhJ>. 
> — You are receiving this because you commented. Reply to this email 
> directly, view it on GitHub < 
> https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65702678>, 
> or mute the thread < 
> https://github.com/notifications/unsubscribe/ABVStiI6V3Tl6KGeXH_el2yb5u5UleY8ks5qICU3gaJpZM4ItfhJ> 
> . -- Godfraind Nicolas 5, rue de la Carrière 6592 Monceau-Imbrechies 
> GSM: 0472 94 80 99 — You are receiving this because you authored the 
> thread. Reply to this email directly, view it on GitHub < 
> https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65704339>, 
> or mute the thread < 
> https://github.com/notifications/unsubscribe/ANpV2jd5pwdhhThgrCfZmtEDH3gRpikCks5qIChXgaJpZM4ItfhJ>. 
> — You are receiving this because you commented. Reply to this email 
> directly, view it on GitHub 
> <https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65705059>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABVSthLUlZB784e_Hj3J_HXlQ5P78jUBks5qICmhgaJpZM4ItfhJ> 
> .
> -- Godfraind Nicolas 5, rue de la Carrière 6592 Monceau-Imbrechies 
> GSM: 0472 94 80 99
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65706148>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ANpV2jZmbYO3NihIqBPaezO4cqkClTMSks5qICtqgaJpZM4ItfhJ>.
>